### DISCUSSION TITLE
Can't move text over image

### DISCUSSION THREAD

**Discussion by andreapx** [ToolJet/ToolJet#6565] [Q&A]
I have a page with an image and I need to place a text over it. I'm able to do that, bu as soon as I release the mouse, I don't see the text box anymore, it's hidden behind the image, but only when I place the mouse over the image, when I'm somewhere else, I can see the text box, so I can't move it.
How can I move it after I've placed it?
Thanks!

**Comment by withshubh** [comment]
By accessing the app builder's left sidebar, you have the option to utilize the `Inspector` tool. Within the component category, you can locate the desired text component. Upon selecting the text component in the inspector, it will also be automatically highlighted on the canvas.

**Reply by andreapx** [reply]
Hi @withshubh I've tried. I can select the component within the inspector, but as soon as I move the mouse away, I lose the selection.
Have a look at the attached video.

https://github.com/ToolJet/ToolJet/assets/13846728/2481f200-e799-4256-b665-2f27c9872923

**Reply by withshubh** [reply]
Hi @andreapx 
Thanks for sharing the video. Seems a valid issue. I'll share this issue with the team.
Could you please share more about the use case? Please check if the [Bounded box](https://docs.tooljet.com/docs/widgets/bounded-box/) would be helpful for you.
